packages/terraform-modules-public/aws/example/variables.tf

```diff
@@ -0,0 +1,215 @@
+variable region {
+    type            = string
+    description     = "AWS Region to provision this cluster"
+    default         = "us-west-1"
+}
+
+variable cidr_blocks {
+    type            = object({
+      vpc                 = string
+      subnet_az1_public   = string
+      subnet_az1_private  = string
+      subnet_az2_public   = string
+      subnet_az2_private  = string
+      allowed_ssh_clients = string
+    })
+    description     = "The cidr_blocks for the different subnets in a redundant Celo network"
+    default         = {
+      vpc                 = "10.10.0.0/16"
+      subnet_az1_public   = "10.10.0.0/24"
+      subnet_az1_private  = "10.10.1.0/24"
+      subnet_az2_public   = "10.10.10.0/24"
+      subnet_az2_private  = "10.10.11.0/24"
+      allowed_ssh_clients = "0.0.0.0/0"
+    }
+}
+
+variable key_pair_name {
+    type            = string
+    description     = "SSH key pair name"
+}
+
+variable celo_image {
+  type              = string
+  description       = "Docker image for Celo nodes"
+  default           = "us.gcr.io/celo-testnet/celo-node:1.8"
+}
+
+variable celo_network_id {
+  type              = string
+  description       = "ID of the Celo network to join"
+  default           = "200110"
+}
+
+variable proxies {
+  description = "Configuration for zero or more proxies in each availability zone."
+  type        = object({
+    az1   = map(object({
+      validator_name              = string
+      validator_signer_address    = string
+    }))
+    az2   = map(object({
+      validator_name              = string
+      validator_signer_address    = string
+    }))
+  })
+  default = {
+    az1     = {}
+    az2     = {}
+  }
+  # Here is an example configuration. We recommend putting this into a secret.auto.tfvars file.
+  # default = {
+  #   az1 = {
+  #     myvalidator_az1_01 = {
+  #       validator_name            = "myvalidator_az1_01"
+  #       validator_signer_address  = "0000000011111111222222223333333344444444"
+  #     }
+  #     myvalidator_az1_02 = {
+  #       validator_name            = "myvalidator_az1_02"
+  #       validator_signer_address  = "5555555566666666777777778888888899999999"
+  #     }
+  #   }
+  #   az2 = {
+  #     myvalidator_az2_01 = {
+  #       validator_name            = "myvalidator_az2_01"
+  #       validator_signer_address  = "4444444433333333222222221111111100000000"
```

Comment:
Why are there multiple validators deployed with different keys?

Reply:
The idea is you can manage a fleet of proxies and validators from this script to manage multiple validators and/or hot standby validators. It's a requirement these would have different signer keys so we don't accidentally double sign. The user would authorize these keys to their validator address when they want to switch which node should sign.

Reply:
I wonder if it'd be simpler to just run the entire script multiple times for each validator set you want to deploy. Given that they have different keys, I view them as different entities altogether. @tkporter what do you think?

Reply:
That is an interesting alternative approach. I decided to model the AWS setup after the GCP setup (it allows multiple validators/proxies), so if we moved to another approach I'd think we'd want to unify it across GCP and AWS in a new PR.

Reply:
Although, I'd need to do more thinking on pros/cons of the approach you're suggesting.

Reply:
Personally I'm fine with the way it is, especially given that it's similar to the GCP setup. I think Terraform is a little painful when it comes to managing multiple deployments and this might encourage hot standby validators. I'm up for bringing others into the loop on this though. @bgeihsgt have you thought any more about this?

Reply:
@jcortejoso do you have opinions?

Reply:
As @tkporter pointed I would say with terraform is easier to manage all your stuff from a single point, although is achievable with workspaces or duplicating 
For me what would makes sense is having a more static package and offer as an fast&easy deployment way (with cloudformation templates for example), and the terraform module offering a more configurable option if people want to customize the deployment a bit more.

Reply:
Anyway I would say that if we think is better to build a "validator stack" module, would be better to address this in a future PR in conjunction with the GCP module.

Reply:
Sounds good, thank all!